csv-data-management/extract_data_from_csv.py

```diff
@@ -337,14 +340,14 @@ def _read_and_return_ds(filepath,params,config):
             # We are passing  specific columns name to filter data from here
             df_file =  _read_file_specific_columns(filepath,columnNames,numberOfRowsToSkip)
             
-        elif isCSVWithoutHeaders: # CSV file without column header and more than one column
-            df_file =  _read_file_no_headers(filepath,numberOfRowsToSkip,noOfColumns) 
-
-        elif isSingleColumn: #CSV with single column and header    
+        elif isSingleColumn and not isCSVWithoutHeaders: #CSV with single column and header    
             df_file = _read_file_single_column(filepath,numberOfRowsToSkip)
         
         elif isSingleColumn and not isCSVWithoutHeaders: # CSV file with one  column and no header 
```

Comment:
These conditions can be further improved to enhance readability

Reply:
There are two if-else block with "isSingleColumn and not isCSVWithoutHeaders:" can you please check if its correct

Reply:
corrected typo